specs/_features/eip7732/beacon-chain.md

```diff
@@ -474,6 +648,138 @@ def process_block(state: BeaconState, block: BeaconBlock) -> None:
 
 #### Withdrawals
 
+##### New `should_process_builder_withdrawal`
+
+```python
+def should_process_builder_withdrawal(
```

Comment:
Not a fan of "should" here. I think I would prefer `is_builder_payment_withdrawable`.